GitHubContents.ps1

```diff
@@ -0,0 +1,93 @@
+function Get-GitHubContents
```

Comment:
You also need to make an update to `PowerShellForGitHub.psd1` and add this file to `NestedModules` and the function to `FunctionsToExport` (both alphabetically please).